string robotName
int16 objarucoid
---
